Compilation Error on Rapberry pi 4

0. Reviewed guide and contributing documents? Yes
1. version [x.y.z, hash, other]: 
2. installed as a package or compiled from sources git 
3. standalone or part of third party motion,
4. video stream source V4L (card or USB), net cam (mjpeg, rtsp, other), mmal
5. hardware ARM
6. operating system Buster

Version PHP | 7.3.9-1~deb10u1
Version OS | Linux raspberrypi 4.19.75-v7l+ #1270 SMP Tue Sep 24 18:51:41 BST 2019 armv7l GNU/Linux [10.1]

Hello,

I have a error with compiler and I don't find a solution
Can you help me please

`gcc -Dsysconfdir=\"/usr/local/etc/motion\" -DLOCALEDIR=\"/usr/local/share/locale\" -g -O2 -D_THREAD_SAFE    -I/usr/include/p11-kit-1   -Irasppicam -I/opt/vc/include  -I/usr/include/arm-linux-gnueabihf    -o motion motion.o logger.o conf.o draw.o jpegutils.o video_loopback.o video_v4l2.o video_common.o video_bktr.o netcam.o netcam_http.o netcam_ftp.o netcam_jpeg.o netcam_wget.o netcam_rtsp.o track.o alg.o event.o picture.o rotate.o translate.o md5.o stream.o ffmpeg.o webu.o webu_html.o webu_stream.o webu_text.o mmalcam.o raspicam/RaspiCamControl.o raspicam/RaspiCLI.o  -pthread   -ljpeg  -lmicrohttpd  -lwebpmux -lwebp  -lmmal_core -lmmal_util -Wl,--push-state,--no-as-needed -lmmal_vc_client -Wl,--pop-state -lvcos -lvchostif -lvchiq_arm -lavutil -lavformat -lavcodec -lswscale -lavdevice
/usr/bin/ld: cannot find -lmmal_core
/usr/bin/ld: cannot find -lmmal_util
/usr/bin/ld: cannot find -lmmal_vc_client
/usr/bin/ld: cannot find -lvcos
/usr/bin/ld: cannot find -lvchostif
/usr/bin/ld: cannot find -lvchiq_arm
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:419: motion] Error 1
make[2]: Leaving directory '/usr/local/src/motion/src'
make[1]: *** [Makefile:549: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/src/motion'
make: *** [Makefile:393: all] Error 2
Making install in src
make[1]: Entering directory '/usr/local/src/motion/src'
gcc -Dsysconfdir=\"/usr/local/etc/motion\" -DLOCALEDIR=\"/usr/local/share/locale\" -g -O2 -D_THREAD_SAFE    -I/usr/include/p11-kit-1   -Irasppicam -I/opt/vc/include  -I/usr/include/arm-linux-gnueabihf    -o motion motion.o logger.o conf.o draw.o jpegutils.o video_loopback.o video_v4l2.o video_common.o video_bktr.o netcam.o netcam_http.o netcam_ftp.o netcam_jpeg.o netcam_wget.o netcam_rtsp.o track.o alg.o event.o picture.o rotate.o translate.o md5.o stream.o ffmpeg.o webu.o webu_html.o webu_stream.o webu_text.o mmalcam.o raspicam/RaspiCamControl.o raspicam/RaspiCLI.o  -pthread   -ljpeg  -lmicrohttpd  -lwebpmux -lwebp  -lmmal_core -lmmal_util -Wl,--push-state,--no-as-needed -lmmal_vc_client -Wl,--pop-state -lvcos -lvchostif -lvchiq_arm -lavutil -lavformat -lavcodec -lswscale -lavdevice
/usr/bin/ld: cannot find -lmmal_core
/usr/bin/ld: cannot find -lmmal_util
/usr/bin/ld: cannot find -lmmal_vc_client
/usr/bin/ld: cannot find -lvcos
/usr/bin/ld: cannot find -lvchostif
/usr/bin/ld: cannot find -lvchiq_arm
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:419: motion] Error 1
make[1]: Leaving directory '/usr/local/src/motion/src'
make: *** [Makefile:549: install-recursive] Error 1`

--- a/configure.ac
+++ b/configure.ac
@@ -217,8 +217,9 @@ AS_IF([test "${MMAL}" = "no"], [
     AC_MSG_CHECKING(for MMAL)
     AS_IF([pkg-config mmal], [
         TEMP_CFLAGS="$TEMP_CFLAGS -Irasppicam "`pkg-config --cflags mmal`
-        #pkg config parm does not provide --no-as-needed variant that buildroot requires
-        TEMP_LIBS="$TEMP_LIBS -lmmal_core -lmmal_util -Wl,--push-state,--no-as-needed -lmmal_vc_client -Wl,--pop-state -lvcos -lvchostif -lvchiq_arm"
+        TEMP_LIBS="$TEMP_LIBS "`pkg-config --libs mmal`
+        #pkg config parm does not provide -lvchostif that buildroot requires
+        TEMP_LIBS="$TEMP_LIBS -lvchostif "
         AC_MSG_RESULT([yes])
         AC_DEFINE([HAVE_MMAL], [1], [Define to 1 if MMAL is around])
       ],[

